docker-java-api/src/main/java/com/github/dockerjava/api/model/SecretSpec.java

```diff
@@ -65,7 +65,7 @@ public String getData() {
      * @see #data
      */
     public SecretSpec withData(String data) {
-        this.data = Base64.encodeBase64String(data.getBytes());
+        this.data = Base64.getEncoder().encodeToString(data.getBytes());
```

Comment:
we had issues with encoders, are they 100% same?

Reply:
if you had issues, why did you add a test? ;)

It should be. Any idea how to test?

Reply:
Not sure with this case, but they are generally not same - with JDK encoder you have better API and you need only to select the right encoder for the purpose. They differ in splitting to lines, for example. See Base64 sources and RFC or write some trivial test to compare.

Btw, problems are usually connected with the usage String.getBytes, because it depends on default system encoding (UTF-8/ISO-8859-2/ISO-8859-1/...?).